torch/nn/init.py

```diff
@@ -105,11 +136,12 @@ def ones_(tensor):
         >>> w = torch.empty(3, 5)
         >>> nn.init.ones_(w)
     """
-    with torch.no_grad():
-        return tensor.fill_(1)
+    return _no_grad_fill_(tensor, 1.)
```

Comment:
Hmm it looks like this is changing the semantics, because previously the type of the tensor would have been preserved, but now it is being filled with floats